<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ebf0f5</string>
  </property>
  <widget class="QPushButton" name="draw_graph">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>640</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>25</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
background-color: #d2e4f4;
color:#46494C;
border-radius:50px;
}

QPushButton:hover{
border:none;
background-color: #adceeb;
color:#46494C;
border-radius:50px;
}</string>
   </property>
   <property name="text">
    <string>Â©</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>655</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>655</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_min">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>657</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fefefe;
border:none;
border-radius:10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="x_max">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>696</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fefefe;
border:none;
border-radius:10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_max">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>697</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fefefe;
border:none;
border-radius:10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_min">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>658</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fefefe;
border:none;
border-radius:10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QCustomPlot" name="q_cutom_plot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
